### PR TITLE
Add AquaSec swag

### LINKED ISSUE #733: AquaSec Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : AquaSec

Q. Where have you seen the announcement (URL)?  
A. : https://blog.aquasec.com/hacktoberfest-2020-aqua-open-source

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing, stickers

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by LeeVaughn**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added information for AquaSec swag to data.json file

closes #733

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -53,6 +53,15 @@
     "dateAdded": "2019-10-07T13:05:05.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "AquaSec",
+    "difficulty": "medium",
+    "description": "In honor of Hacktoberfest, we want to recognize our community and reward your contributions to our open source projects during October.",
+    "reference": "https://blog.aquasec.com/hacktoberfest-2020-aqua-open-source",
+    "image": "https://blog.aquasec.com/hs-fs/hubfs/Hacktoberfest%202.0%20Blog%20Image%20v4.jpg?width=945&name=Hacktoberfest%202.0%20Blog%20Image%20v4.jpg",
```
Comment (aslafy-z) [inline data.json:61]:
```suggestion
    "image": "https://blog.aquasec.com/hs-fs/hubfs/Hacktoberfest%202.0%20Blog%20Image%20v4.jpg",
```

File: data.json [hunk 2]
```diff
@@ -53,6 +53,15 @@
     "dateAdded": "2019-10-07T13:05:05.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "AquaSec",
+    "difficulty": "medium",
+    "description": "In honor of Hacktoberfest, we want to recognize our community and reward your contributions to our open source projects during October.",
```
Comment (aslafy-z) [inline data.json:59]: Can you update the description please?

Comment (LeeVaughn) [reply data.json]: Url should be fixed and description has been updated. Thanks!

File: data.json [hunk 3]
```diff
@@ -53,6 +53,15 @@
     "dateAdded": "2019-10-07T13:05:05.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "AquaSec",
+    "difficulty": "medium",
+    "description": "Based on your contribution, we will be giving out some cool prizes: branded sticker packs, face masks, or limited edition Aqua Hacktoberfest T-shirts!",
```
Comment (aslafy-z) [inline data.json:59]:
```suggestion
    "description": "Contribute to AquaSec's open source portofolio and win branded sticker packs, face masks, or limited edition Aqua Hacktoberfest T-shirts!",
```

**Review by aslafy-z — APPROVED** [review]

**Review by vikaspotluri123 — APPROVED** [review]

**Pull request merged**